conda_docker/conda.py

```diff
@@ -647,11 +685,21 @@ def build_docker_environment(
     with timer(LOGGER, "writing docker file"):
         image.write_filename(output_filename)
 
+def copy_pip_packages(target_dir, pip_files):
+    pip_targ_dir = os.path.join(target_dir, "opt/", "conda/")
+    for pip_file in pip_files:
+        pip_file_path_ending = pip_file[pip_file.index('lib'):pip_file.rindex('/')]
```

Comment:
I'm a little confused what's going on here. In particular, why `lib`. Maybe some commenting would help, but how about a quick zoom after standup. You might have explained this before but I could use a refresher